Regression/WarpX-tests.ini

```diff
@@ -2310,3 +2310,111 @@ numthreads = 0
 compileTest = 0
 doVis = 0
 analysisRoutine = Examples/Tests/ParticleDataPython/analysis.py
+
+[Performance_works_1_uniform_rest_32ppc]
+buildDir = .
+inputFile = Examples/Tests/PerformanceTests/automated_test_1_uniform_rest_32ppc
+runtime_params = amr.n_cell=64 64 64 max_step=10
+dim = 3
+addToCompileString =
+restartTest = 0
+useMPI = 1
+numprocs = 2
+useOMP = 0
+numthreads = 0
+compileTest = 0
+doVis = 0
+compareParticles = 1
+particleTypes = electrons ions
+analysisRoutine =
+tolerance = 1.0e-4
+
+[Performance_works_2_uniform_rest_1ppc]
+buildDir = .
+inputFile = Examples/Tests/PerformanceTests/automated_test_2_uniform_rest_1ppc
+runtime_params = amr.n_cell=64 64 64 max_step=10
+dim = 3
+addToCompileString =
+restartTest = 0
+useMPI = 1
+numprocs = 2
+useOMP = 0
+numthreads = 0
+compileTest = 0
+doVis = 0
+compareParticles = 1
+particleTypes = electrons
+analysisRoutine =
+tolerance = 1.0e-4
+
+[Performance_works_3_uniform_drift_4ppc]
+buildDir = .
+inputFile = Examples/Tests/PerformanceTests/automated_test_3_uniform_drift_4ppc
+runtime_params = amr.n_cell=64 64 64 max_step=10
+dim = 3
+addToCompileString =
+restartTest = 0
+useMPI = 1
+numprocs = 2
+useOMP = 0
+numthreads = 0
+compileTest = 0
+doVis = 0
+compareParticles = 1
+particleTypes = electrons ions
+analysisRoutine =
+tolerance = 1.0e-4
+
+[Performance_works_4_labdiags_2ppc]
+buildDir = .
+inputFile = Examples/Tests/PerformanceTests/automated_test_4_labdiags_2ppc
+runtime_params = amr.n_cell=64 64 64 max_step=10
+dim = 3
+addToCompileString =
+restartTest = 0
+useMPI = 1
+numprocs = 2
+useOMP = 0
+numthreads = 0
+compileTest = 0
+doVis = 0
+compareParticles = 1
+particleTypes = electrons ions
+analysisRoutine =
+tolerance = 1.0e-4
+
+[Performance_works_5_loadimbalance]
+buildDir = .
+inputFile = Examples/Tests/PerformanceTests/automated_test_5_loadimbalance
+runtime_params = amr.n_cell=64 64 64 max_step=10
+dim = 3
+addToCompileString =
+restartTest = 0
+useMPI = 1
+numprocs = 2
+useOMP = 0
+numthreads = 0
+compileTest = 0
+doVis = 0
+compareParticles = 1
+particleTypes = electrons ions
+analysisRoutine =
+tolerance = 1.0e-4
+
+[Performance_works_6_output_2ppc]
+buildDir = .
+inputFile = Examples/Tests/PerformanceTests/automated_test_6_output_2ppc
```

Comment:
Challenge: all performance tests but this file do not have an output and this have an unused variable warning from the `.ini` files injected `plot_file_name`.
unused variables are an error in CI.

Trying to overwrite test 1-5 with `plot_file_name =` now.

Reply:
Nope, does not work. Will need to append to runtime params the block `diagnostics.diags_names=diag1 diag1.intervals=0 diag1.diag_type=Full` or so